Formula/guile@2.rb

```diff
@@ -0,0 +1,58 @@
+class GuileAT2 < Formula
```

Comment:
This seems like a possible candidate to test the new depreciation DSL if we have that info.

Reply:
What information is needed for this?

Reply:
I didn't look at the DSL yet. But I thought it needed a date. @MikeMcQuaid do you have an example of the deprecation DSL?

Reply:
Is it the case we deprecate _every_ versioned formula or what is the line here? I ask because Guile 2.2 is still supported.

Reply:
My suggestion would be adding a deprecation to every versioned formula with a known EOL date.

Reply:
`deprecate! :date => "01-03-2020"` (or any date it can parse). The `:date` is optional. There's also `disable!`

Reply:
> any date it can parse

dd-mm-yyyy or mm-dd-yyyy?

Reply:
https://xkcd.com/1179/

Reply:
@Bo98 Try it and see 😂.